src/github.com/getlantern/lantern-ui/app/js/app.js

```diff
@@ -122,15 +122,20 @@ var app = angular.module('app', [
           return;
         }
 
-        // try to reconnect indefinitely
-        // when the websocket closes
-        $interval(function() {
-          console.log("Trying to reconnect to disconnected websocket");
-          ds = $websocket('ws://' + document.location.host + '/data');
-          ds.onOpen(function(msg) {
-            $window.location.reload();
-          });
-        }, WS_RECONNECT_INTERVAL);
+        // Temporary workaround for https://bugzilla.mozilla.org/show_bug.cgi?id=1192773
+        if (navigator.userAgent.toLowerCase().indexOf('firefox') > -1) {
+          $rootScope.backendIsGone = true;
+          $rootScope.$digest()
+        } else {
+          // Try to reconnect indefinitely when the websocket closes
+          $interval(function() {
```

Comment:
The interval-triggered function is only executed in browsers that are not Firefox. That is, when we are on Firefox, we won't reconnect and it will show the localized "Lantern Gone" message.